app/Http/Controllers/TagController.php

```diff
@@ -181,4 +189,51 @@ public function entityStore(StoreTagEntity $request, Tag $tag)
         return redirect()->route('tags.show', $redirectUrlOptions)
             ->with('success', trans('tags.children.create.success', ['name' => $tag->name]));
     }
+
+    /**
+     * @param Tag $tag
+     * @return \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function transferTag(Tag $tag)
+    {
+        $this->authorize('update', $tag);
+
+        $models = Tag::where('campaign_id', $tag->campaign_id)->get();
+
+        $tags[''] = __('tags.transfer.placeholder');
+
+        foreach ($models as $model) {
+            if ($tag->id != $model->id) {
```

Comment:
Several campaigns have over 1000 tags. We need to use the ajax dropdown, otherwise we'll crash the server and kill our bandwidth.